src/www/ui/api/documentation/openapi.yaml

```diff
@@ -3747,6 +3787,38 @@ components:
           type: boolean
           description: Add the copyright information as text findings
           default: false
+    MutateStdComment:
+      type: object
+      properties:
+        commentId:
+          type: integer
+          description: |
+            The ID of the standard comment.
+            Required if update is TRUE.
+          example: 2
+        name:
+          type: string
+          description: |
+            The name of the standard comment.
+            Not required if update is TRUE and toggle is set.
+          example: commentName
```

Comment:
IMO, name should only be required if user wants to update the name. If the user is providing ID for the comment, name is just an additional data which can be retrieved from DB unambiguously.

Reply:
Hope things look great now